<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuXianxianshi</class>
 <widget class="QWidget" name="QuXianxianshi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>451</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
